### DISCUSSION TITLE
Cluster function or a function to connect multiple servers and run in one system

### DISCUSSION THREAD

**Discussion by Tizian-Maxime-Weigt** [ZoeyVid/NPMplus#373] [Q&A] [answered]
A function with which you can synchronize servers with each other or connect them to form a cluster

**Comment by Zoey2936** [comment]
so you mean you make changes on npm1 and npm2 copies them?

**Reply by Tizian-Maxime-Weigt** [reply]
yeah also that I can connect multiple servers ...

**Reply by Tizian-Maxime-Weigt** [reply]
i have test it with nfs but this does not work ...

**Reply by Zoey2936** [reply]
what do you mean with "that I can connect multiple servers"

**Reply by Tizian-Maxime-Weigt** [reply]
Haha da du ja Deutsch kannst schreibe ich einfach in Deutsch also ich suche bzw. schreibe eine NPM version die halt eine Cluster Funktion hat also ein master hat und dann sozusagen die slaves das alles übernehmen so das man theoretisch alles ddos sicher machen kann bzw es einfach ausfallsicher ist.

**Reply by Zoey2936** [reply]
aber reicht es dazu nicht das ganze in einen netzwerk speicher zu mounten also /data und dann wird das alles ja automatisch kopiert und sowieso stündlich neugeladen

**Reply by Tizian-Maxime-Weigt** [reply]
ne leider geht das nicht also habe es getestet geht ned... kannst ja gerne selber mal machen kannst ja mal Discord annehmen

**Reply by Zoey2936** [reply]
was geht denn genau nicht?

**Reply by Tizian-Maxime-Weigt** [reply]
Der Container startet erst gar ned auch wenn man via Portainer in den Logs bzw in der Console schaut geht es ned habe auch schon das ohne Docker Probiert also alles Komplett ohne Container in einem LXC getestet tcha geht also leider ned anderes als sowas zu implementieren :/

**Reply by Zoey2936** [reply]
wie hast du den netzwerkspeicher gemounted?

**Reply by Tizian-Maxime-Weigt** [reply]
normal also rw und alle rechte und so also habe es gemacht wie man es halt immer macht :c

**Reply by Zoey2936** [reply]
ne, ich meine über docker selbst oder fstab etc.

**Reply by Tizian-Maxime-Weigt** [reply]
Habe ich ja getestet und dann steht immer sowas wie keine rechte oder kann nicht starten oder was war's ja ka jetzt mehr auf jeden Fall geht es Ned... Und wenn man sowas implementiert kann man au failover und so einbauen wäre auch besser :)

**Reply by Zoey2936** [reply]
kann ich mal die config sehen?

**Reply by Tizian-Maxime-Weigt** [reply]
Add mich Mal bei Discord :c tizian156

**Reply by Zoey2936** [reply]
ich bevorzuge ehrlich gesagt github sachen auch über github zu lösen

**Reply by Tizian-Maxime-Weigt** [reply, marked as answer]
Problem hat sich geklärt habe mir eine sync option selber eingebaut geht jetzt wie ich das will ;D danke trzd

**Reply by Zoey2936** [reply]
ok
